src/test/components/Card.test.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+
+import Card from '../../components/Card';
+
+const testItem = {
+  snippet: {
+    title: 'Video Test',
+    description: 'Video test text',
+    thumbnails: {
+      default: {
+        url: 'https://i.ytimg.com/vi/nmXMgqjQzls/default.jpg',
+        width: 120,
+        height: 90,
+      },
+      medium: {
+        url: 'https://i.ytimg.com/vi/nmXMgqjQzls/mqdefault.jpg',
+        width: 320,
+        height: 180,
+      },
+      high: {
+        url: 'https://i.ytimg.com/vi/nmXMgqjQzls/hqdefault.jpg',
+        width: 480,
+        height: 360,
+      },
+    },
+  },
+};
+
+beforeEach(() => {
+  render(<Card item={testItem} />);
+});
+
+describe('Card', () => {
+  it('renders title, text and image', () => {
+    const titleElement = screen.queryByText('Video Test');
```

Comment:
It is recommended to not use `queryBy` selectors when not testing for not existence

Source on [Common Mistakes using React Testing Library](https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#using-query-variants-for-anything-except-checking-for-non-existence)